File: Balanced Number (Special Numbers Series #1 )/Balanced Number (Special Numbers Series #1 ).py
def balanced_num(number):
    num = str(number)
    print(number)
    if len(num)<=2:
        return "Balanced"
    else:
        if len(num)%2==1:
            return "Balanced" if sum(map(int,list(num[:len(num)//2])))==sum(map(int,list(num[len(num)//2+1:]))) else "Not Balanced"
        else:
            return "Balanced" if sum(map(int,list(num[:len(num)//2-1])))==sum(map(int,list(num[len(num)//2+1:]))) else "Not Balanced"
        
        

File: Balanced Number (Special Numbers Series #1 )/Balanced Number (Special Numbers Series #1 ).dart
String balancedNum(int number) {
  String num = number.toString();
  print(number);
  if (num.length <= 2) {
    return "Balanced";
  } else {
    if (num.length % 2 == 1) {
      int leftSum = num.substring(0, num.length ~/ 2).split('').map(int.parse).reduce((a, b) => a + b);
      int rightSum = num.substring(num.length ~/ 2 + 1).split('').map(int.parse).reduce((a, b) => a + b);
      return leftSum == rightSum ? "Balanced" : "Not Balanced";
    } else {
      int leftSum = num.substring(0, num.length ~/ 2 - 1).split('').map(int.parse).reduce((a, b) => a + b);
      int rightSum = num.substring(num.length ~/ 2 + 1).split('').map(int.parse).reduce((a, b) => a + b);
      return leftSum == rightSum ? "Balanced" : "Not Balanced";
    }
  }
}
